package.json

```diff
@@ -1,9 +1,22 @@
 {
   "name": "@lokalise/your-project-name",
-  "description": "Template repository for creating NPM packages",
+  "version": "1.0.0",
   "scripts": {
-    "build": "echo TODO",
-    "test": "echo TODO",
-    "lint": "echo TODO"
+    "build": "vite build"
+  },
+  "type": "module",
+  "files": [
```

Comment:
should we include readme too?

Reply:
README is being handled in a future ticket.

Reply:
I mean, publish it too, but sure!

Reply:
O sorry I see what you meant!

The README does already get published. You can see it in the dryrun output image I posted on the PR description.

It works because `npm publish` automagically looks for README and package.json in the current working directory _regardless of whatever additional options you provide for publishing_

So you should think of the `files` here as _in addition to_ README and package.json